utils/mt5311/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mt5311
+PKG_VERSION:=238-2cb1304
+PKG_RELEASE:=1
+
+PKG_LICENSES:=AGPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+PKG_URL:=https://github.com/jimdigriz/mt5311
```

Comment:
This is already at line 24. No need to have it twice.

Reply:
fixed.